compiler/modules/nback/build.bal

```diff
@@ -2103,15 +2109,15 @@ final readonly & record {|
     { domain: t:TOP, repr: REPR_TOP }
 ];
 
-function semTypeRetRepr(t:SemType ty) returns RetRepr|BuildError {
+function semTypeRetRepr(t:SemType ty, err:File file) returns RetRepr|BuildError {
     if ty === t:NIL {
         return REPR_VOID;
     }
-    return semTypeRepr(ty);
+    return semTypeRepr(ty, file);
 }
 
 // Return the representation for a SemType.
-function semTypeRepr(t:SemType ty) returns Repr|BuildError {
+function semTypeRepr(t:SemType ty, err:File file) returns Repr|BuildError {
```

Comment:
If you add a parameter, then add something that can give the proper location e.g. a ModuleLevelDefn (which you can use with s:defnLocation

Reply:
Changed the parameter type to `err:Location` and had callers get it from scaffold or using the parameters to passed to the constructor in the case of scaffold's init function

Reply:
No. That's not what I said.